### DISCUSSION TITLE
Worse free version with installer

### DISCUSSION THREAD

**Discussion by Ihateyoudattebayo** [mafiosnik777/enhancr#10] [General]
When will there be a worse version on Github which uses the installer, instead of hand building the python environment,

**Comment by mafiosnik777** [comment]
@Ihateyoudattebayo A free version will be made available on 1.0 release, so 4 more versions from now. 
If it's gonna be worse or with some features limited I can't say, maybe just 3 versions behind or something like that. Not quite sure about that, yet.

**Reply by Ihateyoudattebayo** [reply]
That sounds pretty good.

**Reply by Ihateyoudattebayo** [reply]
> @Ihateyoudattebayo A free version will be made available on 1.0 release, so 4 more versions from now. If it's gonna be worse or with some features limited I can't say, maybe just 3 versions behind or something like that. Not quite sure about that, yet.

that was a lie.


You released it on 0.98

**Reply by mafiosnik777** [reply]
💀

I swapped around the roadmap from free release & cross-platform release so people don't use stuff like waifu2x-extension-gui and trash their SSDs with frame extraction, because it just uses nihui binaries.

**Reply by Ihateyoudattebayo** [reply]
> 💀
> 
> I swapped around the roadmap from free release & cross-platform release so people don't use stuff like waifu2x-extension-gui and trash their SSDs with frame extraction, because it just uses nihui binaries.

real
